src/Disclosure/interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface DisclosureProps {
+  title: string
+  content: string
```

Comment:
`content` lebih baik jadi `children` aja, karena gk menutup kemungkinan kalo disclosure itu kontennya image atau rich text

```suggestion
  children: ReactNode
```

Reply:
done